README.md

```diff
@@ -0,0 +1,60 @@
+# LeaderBoard
+
+> This is a real time Leaderboard App which manage score and provides the latest data from acsending order. Enjoy Watching!
+
+## Live Demo
+
+- Demo Link: [@leaderboard](https://hector096.github.io/Leaderboard/dist)
```

Comment:
- [ ] To make the README professional. Please update the Live Demo Link to the correct one. For now, the link is broken.